core/services/ardupilot_manager/setup.py

```diff
@@ -13,5 +13,11 @@
     long_description_content_type="text/markdown",
     packages=setuptools.find_packages(),
     python_requires=">=3.6",
-    install_requires=["appdirs==1.4.4", "packaging==20.4", "smbus2==0.3.0"],
```

Comment:
The commit message does no help, which setup this commits points to ? please add the service name as prefix